tests/jballerina-unit-test/src/test/resources/test-src/error/error_destructure_field_binding_negative.bal

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+type FooError error<record {|string message?; string cause; boolean fatal?; string...; |}>;
```

Comment:
```suggestion
type FooError error<record {| string message?; string cause; boolean fatal?; string...; |}>;
```

Reply:
There is an issue with the formatter. #34023. I think the correct formatting is without whitespace. Will update.

```suggestion
type FooError error<record {|string message?; string cause; boolean fatal?; string...;|}>;
```